lib/grn_ecmascript.lemon

```diff
@@ -417,9 +417,18 @@ primary_expression ::= STRING.
 primary_expression ::= BOOLEAN.
 primary_expression ::= NULL.
 
+
 array_literal ::= BRACKETL elision BRACKETR.
 array_literal ::= BRACKETL element_list elision BRACKETR.
 array_literal ::= BRACKETL element_list BRACKETR.
+array_literal ::= BRACKETL BRACKETR. {
```

Comment:
Can we use existing `BRACKETL elision BRACKETR` for this instead of creating a new rule?

Ah, existing `array_literal` definition isn't good.

It should be:

```text
array_literal ::= BRACKETL element_list BRACKETR.

element_list ::= .
element_list ::= assignment_expression.
element_list ::= element_list COMMA assignment_expression.
```